doc/adr/0002-use-meta-yaml-to-track-feedstock-metadata.md

```diff
@@ -69,21 +69,21 @@ recipes:
 
 Some feedstocks may want to provide many recipes in one `.py` file.
 In this case, it is not feasible to enumerate every one explicitly in meta.yaml.
-Instead, we can point at a python dict directly using `dict-object`.
+Instead, we can point at a python dict directly using `dict_object`.
 The keys of the dict will become the recipe ids, and the values the recipe objects.
 For example:
 
 ```yaml
 recipes:
-  - dict-object: "recipe:all_recipes
+  - dict_object: "recipe:all_recipes
```

Comment:
@sharkinsspatial, do I understand correctly that, in the mapping `"recipe:all_recipes"`, `recipe` refers to the name of the module containing the dictionary (in this case, `recipe.py`), and `all_recipes` is the name of the dictionary object within that module?